src/test/java/uk/gov/hmcts/reform/civil/handler/callback/user/EvidenceUploadHandlerTest.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.reform.civil.handler.callback.user;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.SubmittedCallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CallbackType;
+import uk.gov.hmcts.reform.civil.handler.callback.BaseCallbackHandlerTest;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+import uk.gov.hmcts.reform.civil.sampledata.CaseDataBuilder;
+
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_START;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(classes = {
+    EvidenceUploadHandler.class,
+    JacksonAutoConfiguration.class
+})
+public class EvidenceUploadHandlerTest extends BaseCallbackHandlerTest {
+
+    @Autowired
+    private EvidenceUploadHandler handler;
+
+    @Nested
+    class AboutToStartCallback {
+
+        @Test
+        void aboutToStartCallback_placeholder() {
+            CaseData caseData = CaseDataBuilder.builder().build();
+            CallbackParams params = callbackParamsOf(caseData, ABOUT_TO_START);
+
+            AboutToStartOrSubmitCallbackResponse response = (AboutToStartOrSubmitCallbackResponse) handler
+                .handle(params);
+
+        }
```

Comment:
What's being tested here?

I don't see any assertion or anyway anything that could give a clue as to when the test is succeeding or failing.

Same for the other tests in this class.

Please clarify in the code what defines the test as successful. If assertions don't apply, put an inline comment of sorts.